### PR TITLE
Eliminate magic number 8 and add note for capacity growth strategy.

### PR DESCRIPTION
**Pull request by yongw5**
Eliminating magic number to impove readable.

@veblush

### CODE REVIEW

**Comment by veblush** [comment]
@yongw5 Good catch! LGTM.

**Review by veblush — APPROVED** [review]

**Comment by veblush** [comment]
@yongw5 Would you sign-up EasyCLA so that we can merge this?

File: src/core/lib/iomgr/timer_heap.cc [hunk 1]
```diff
@@ -96,6 +96,8 @@ void grpc_timer_heap_destroy(grpc_timer_heap* heap) { gpr_free(heap->timers); }
 
 bool grpc_timer_heap_add(grpc_timer_heap* heap, grpc_timer* timer) {
   if (heap->timer_count == heap->timer_capacity) {
+   /* Capacity growth strategy is 1.5x. See FBVector.md for details.
```
Comment (veblush) [inline src/core/lib/iomgr/timer_heap.cc:99]: I don't have context here but how do you this is related with FBVector?

**Comment by yongw5** [comment]
@veblush EasyCLA is signed.

**Comment by yongw5** [comment]
@ctiller Comment linking to FBVector is deleted.

**Pull request closed without merging**